README.md

```diff
@@ -295,5 +310,34 @@ module.exports = {
 };
 ```
 
+### Wallet
+To configure a wallet for your project, specify it by using `wallets["walletName"]`.
+You can specify multiple wallets/accounts.
+
+The parameters for the wallet are:
+  - `accountName`: The name to give the account. If omitted, the default value `__default__ ` will be used;
+  - `modulePath`: The python module and wallet class of your chosen wallet provider;
+  - `accountPath`: The path where your wallet information will be saved.
+
+```javascript
+module.exports = {
+  ...
```

Comment:
I remember putting the ellipsis before and after blocks when showing example usage, but ended up having only the one after the block.